### PR TITLE
[next] Sprite, Mesh and Graphics refactor to including in batching

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
Graphics and Meshes are not batched if they are not too complex (< 100 points)

This gives up to 5x better perf on graphics and meshes.

### CODE REVIEW

**Comment by bigtimebuddy** [comment]
@GoodBoyDigital could you resolve conflicts and make sure Travis passes? Looks like some linting issues preventing building and tests from running.

**Comment by GoodBoyDigital** [comment]
ah no worries!

File: packages/display/src/DisplayObject.js [hunk 1]
```diff
@@ -115,6 +115,25 @@ export default class DisplayObject extends EventEmitter
         this._mask = null;
 
         /**
+<<<<<<< HEAD
```
Comment (kperwel) [inline packages/display/src/DisplayObject.js]: Unresolved conflict hidden in comments.

File: packages/graphics/src/utils/buildCircle.js [hunk 2]
```diff
@@ -13,85 +11,70 @@ import { hex2rgb } from '@pixi/utils';
...
+        const y = circleData.y;
+        let width;
+        let height;
+
+        points.length = 0;
+
+        // TODO - bit hacky??
+        if (graphicsData.type === SHAPES.CIRC)
+        {
+            width = circleData.radius;
```
Comment (kperwel) [inline packages/graphics/src/utils/buildCircle.js]: shouldn't be 2*radius? (just guessing)

**Comment by bigtimebuddy** [comment]
This branch is compatible with **dev**: https://pixijs.io/v5-render-tests/?feature/mega-batch (100%)

**Pull request merged**